x-pack/plugin/security/qa/profile/build.gradle

```diff
@@ -12,9 +12,7 @@ dependencies {
   javaRestTestImplementation project(':x-pack:plugin:security')
 }
 
-boolean literalUsername = buildParams.random.nextBoolean()
```

Comment:
buildParams.random resolves the TestSeed and breaks configuration-cache. by moving this into the task configuration block we defer that. for this particular test cc will always be out of date. Longterm we might wand a more deterministic way of testing this behaviour IMO

Reply:
The altenative would be to split the random from the testseed,, but this would break reproducability.